migrations/V153__Add_Primary_Point_Of_Contact_Configuration.sql

```diff
@@ -0,0 +1,54 @@
+ALTER TABLE possible_operational_solution_contact
+  ADD COLUMN is_primary BOOLEAN DEFAULT NULL;

```

Comment:
I think it might be good to make this `NOT NULL` and just have it be explicitly `true` or `false`, since I don't think we have a reason to differentiate `NULL` and `false`.